src/comitClient.ts

```diff
@@ -8,7 +8,7 @@ export class ComitClient {
   constructor(
     private readonly bitcoinWallet: BitcoinWallet,
     private readonly ethereumWallet: EthereumWallet,
-    private readonly cnd: Cnd
+    public readonly cnd: Cnd
```

Comment:
:question: I think you are only doing this to get the peer ID and the address hint. What if `ComitClient` handled getting this for you instead?

Reply:
Ok.